packages/expo-notifications/android/src/main/java/expo/modules/notifications/notifications/NotificationSerializer.java

```diff
@@ -29,6 +33,9 @@ public static Bundle toBundle(NotificationResponse response) {
     Bundle serializedResponse = new Bundle();
     serializedResponse.putString("actionIdentifier", response.getActionIdentifier());
     serializedResponse.putBundle("notification", toBundle(response.getNotification()));
+    if (response instanceof TextInputNotificationResponse) {
+      serializedResponse.putString("userText", ((TextInputNotificationResponse)response).getUserText());
```

Comment:
this doesn't seem formatted 💅

Reply:
👍  Need to get into the habit of formatting with Android Studio (unless you've found a better way)